src/iks001_computePi.cpp

```diff
@@ -8,12 +8,62 @@
 #include <dune/alugrid/grid.hh>
 #include <dune/geometry/quadraturerules.hh>
 #include <dune/grid/common/boundarysegment.hh>
+#include <dune/grid/io/file/vtk/vtkwriter.hh>
 
 #include <Eigen/Core>
 #include <Eigen/Dense>
 
 #include <ikarus/utils/drawing/griddrawer.hh>
 
+void globallyRefinedUnitCircle() {
```

Comment:
Can you rename this from globa.. to something like naive or citrcle unaware refinement. I think local and global are unapproriate identifiers here.

Reply:
renamed to boundaryUnawareRefinedCircle()